src/com/pyler/xinstaller/Preferences.java

```diff
@@ -39,6 +39,7 @@ protected void onCreate(Bundle savedInstanceState) {
 		@Override
 		public void onCreate(Bundle savedInstanceState) {
 			super.onCreate(savedInstanceState);
+            context = getActivity();
```

Comment:
Right now I am cleaning up the preferences. So far I moved the preference
keys to Common and I am working on moving to preference headers.
On Oct 18, 2015 4:21 PM, "Devin Hussey" <husseydevin@gmail.com> wrote:

> Sorry I use AIDE and it doesn't learn formatting. Will fix.
> On Oct 18, 2015 4:19 PM, "pyler" <notifications@github.com> wrote:
>
>> In src/com/pyler/xinstaller/Preferences.java
>> <https://github.com/pylerSM/XInstaller/pull/27#discussion_r42325772>:
>>
>> > @@ -39,6 +39,7 @@ protected void onCreate(Bundle savedInstanceState) {
>> >  		@Override
>> >  		public void onCreate(Bundle savedInstanceState) {
>> >  			super.onCreate(savedInstanceState);
>> > +            context = getActivity();
>>
>> add space please
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/pylerSM/XInstaller/pull/27/files#r42325772>.
>>
>